content/en/docs/tutorials/add-prebuilt-driver-config/index.md

```diff
@@ -0,0 +1,130 @@
+# Add prebuilt drivers for a kernel release
+
+## Requirements
+
+- Ensure the Linux distribution that distributes the Linux kernel version is supported by [driverkit](https://github.com/falcosecurity/driverkit/blob/master/docs/driverkit.md).
+- Ensure the driver is not already available for the specific Linux kernel version, by checking it at [download.falco.org/driver/site](https://download.falco.org/driver/site/index.html).
+
+## Generate the configuration
+
+You can generate and contribute configurations for your OS, as follows.
+
+Let's start by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [test-infra](https://github.com/falcosecurity/test-infra) repository.
+
+For example using the [GitHub CLI](https://cli.github.com/), run:
+
+```shell
+gh repo fork falcosecurity/test-infra
+```
+
+From the directory of the local repository directory, create a dedicated Git branch:
+
+```shell
+cd test-infra
+git checkout -b new/driver-build-config
+```
+
+and run:
+
+```shell
+make -C driverkit generate -e TARGET_DISTRO=arch -e TARGET_KERNEL_RELEASE=<release name> -e TARGET_KERNEL_VERSION=<build version>
+```
+
+filling it with:
+* `TARGET_DISTRO`: the Linux distribution of the target host.
+* `TARGET_KERNEL_RELEASE`: the Linux kernel release name of the target host. You can fill it with the output of the command `"$(uname -r)"`, running it on a shell on the target host.
+* `TARGET_KERNEL_VERSION`: the Linux kernel build version of the target host. You can fill it with output of the command: `"$(uname -v | sed 's/#\([[:digit:]]\+\).*/\1/')"`, running it on a shell on the target host.
+
+for example:
+
+```shell
+make -C driverkit generate -e TARGET_DISTRO=arch -e TARGET_KERNEL_RELEASE=6.3.4-arch1-1 -e TARGET_KERNEL_VERSION=1
+```
+
+> Available values for `$TARGET_DISTRO` can be found [here](https://github.com/falcosecurity/driverkit/blob/master/docs/driverkit.md).
+
+### Advanced settings
+
+> Besides `$TARGET_DISTRO`, `$TARGET_KERNEL_RELEASE` and `TARGET_KERNEL_VERSION`, you can find all the supported filters [here](https://github.com/falcosecurity/test-infra/blob/master/driverkit/README.md#available-make-targets), that help you to target specific kernel releases.
+
+## Propose the configuration
```

Comment:
I'd move this section to much later, once the user understand the file they have generated, as well as it's been validated.

Reply:
Thank you @vjjmiras. I've just moved it, let me know if it aligns with what you had in mind

Reply:
I misunderstood your point @vjjmiras. Moved now the _Propose the configuration_ section after the validation one.